sdk/node/test/accessTokens.js

```diff
@@ -0,0 +1,139 @@
+/* eslint-env mocha */
+
+const chain = require('../dist/index.js')
+const uuid = require('uuid')
+const chai = require('chai')
+const chaiAsPromised = require('chai-as-promised')
+
+chai.use(chaiAsPromised)
+const expect = chai.expect
+
+const client = new chain.Client()
+const tokenId = `token-${uuid.v4()}`
+const clientTokenId = `client-${uuid.v4()}`
+const networkTokenId = `network-${uuid.v4()}`
+
+describe('Access tokens test', () => {
+
+  describe('Promise style', () => {
+
+    describe('Access token creation', () => {
+
+      it('access token creation successful', () => {
+        return client.accessTokens.create({
```

Comment:
I can go back and forth on this, but I'd like to lean heavily towards always* having an explicit assertion in test cases? For example, in the [Ruby test for access token creation](https://github.com/chain/chain/blob/main/sdk/ruby/spec/integration/access_tokens_spec.rb#L4-L6), we test that we get a token value back. 

\* Excepting the callback tests, where we've accepted that the implicit assertion of "success" is good enough.